SignalMessaging/profiles/OWSUserProfile.m

```diff
@@ -106,18 +106,17 @@ - (void)setAvatarUrlPath:(nullable NSString *)avatarUrlPath
 {
     @synchronized(self)
     {
-        BOOL didChange;
-        if (_avatarUrlPath == nil && avatarUrlPath == nil) {
-            didChange = NO;
-        } else if (_avatarUrlPath != nil || avatarUrlPath != nil) {
```

Comment:
I could have also just changed this line to:
```
} else if (_avatarUrlPath == nil || avatarUrlPath == nil) {
```

But since we got it wrong... twice... I rethought this idiom.

Reply:
Funny, I applied a fix for this in the backup work...